packages/react-components/theme-designer/src/components/Demo/Demo.tsx

```diff
@@ -0,0 +1,222 @@
+import * as React from 'react';
+import { makeStyles, mergeClasses } from '@griffel/react';
+import {
+  FluentProvider,
+  teamsLightTheme,
+  Body1,
+  Title3,
+  TabList,
+  Tab,
+  Input,
+  Button,
+  Label,
+  Menu,
+  MenuTrigger,
+  MenuList,
+  MenuButton,
+  MenuItemCheckbox,
+  MenuPopover,
+  Slider,
+  Badge,
+  Switch,
+  Radio,
+  Checkbox,
+  Avatar,
+} from '@fluentui/react-components';
+import {
+  SearchRegular,
+  bundleIcon,
+  CutRegular,
+  CutFilled,
+  ClipboardPasteRegular,
+  ClipboardPasteFilled,
+  EditRegular,
+  EditFilled,
+  ChevronRightRegular,
+  MeetNowRegular,
+  MeetNowFilled,
+  CalendarLtrFilled,
+  CalendarLtrRegular,
+} from '@fluentui/react-icons';
+
+export interface ContentProps {
+  className?: string;
+}
+
+const useStyles = makeStyles({
+  root: {
+    display: 'grid',
+    alignItems: 'start',
+    justifyContent: 'center',
+    gridTemplateColumns: 'repeat(3, 1fr)',
+    gridColumnGap: '5%',
+  },
+  col1: {
+    display: 'grid',
+    gridTemplateRows: 'repeat(3, auto)',
+    gridRowGap: '10%',
+    justifyContent: 'center',
+    alignItems: 'stretch',
+  },
+  col2: {
+    display: 'grid',
+    gridTemplateRows: 'repeat(3, 1fr)',
+    gridRowGap: '10%',
+    justifyContent: 'center',
+    alignItems: 'stretch',
+  },
+  col3: {
```

Comment:
this right cell looks like a stack of horizontal flex containers

![image](https://user-images.githubusercontent.com/1434956/170539877-6376b9ce-86fd-4cb1-ad00-aa3c75dd8f21.png)


not a huge deal, but try to avoid applying styles directly to slider, button etc. use the parent to lay things out.